src/App.js

```diff
@@ -36,7 +38,7 @@ class App extends Component {
     this.state = {
       langCode: `en`,
       snippets: getTextForLanguage(`en`),
-      clients: {
+      clients:  {
```

Comment:
Tiny linting error here!

Reply:
Dangit! I ran the lint command and got no changes!

Reply:
Oh, no, this is correct. It lines up with the previous key/value pair.

Reply:
Oh your're right, sorry! I saw I had a lint error there currently and for some reason assumed this was causing the problem instead of fixing it (doesn't really make sense now that I think about it...)